x/auth/tx/service_test.go

```diff
@@ -448,7 +448,7 @@ func (s IntegrationTestSuite) mkTxBuilder() client.TxBuilder {
 		WithSignMode(signing.SignMode_SIGN_MODE_DIRECT)
 
 	// Sign Tx.
-	err := authclient.SignTx(txFactory, val.ClientCtx, val.Moniker, txBuilder, false)
+	err := authclient.SignTx(txFactory, val.ClientCtx, val.Moniker, txBuilder, false, true)
```

Comment:
If we choose default behavior as false, we should be consistent with it being `false` default unless we are explicitly trying to test for overwrite behavior.

Reply:
There is no default here. Previously this function was used as `overwrite=true` everywhere. I didn't change it on the `SignTx` nor `tx.Sign` level.

Reply:
Ok, fair. let's keep with pre-existing behavior.